### PR TITLE
set_image now can accept individual parameters

### PR DESCRIPTION
**Pull request by zerinrc**

### CODE REVIEW

File: reolink_aio/api.py [hunk 1]
```diff
@@ -3165,6 +3165,59 @@ async def set_ai_sensitivity(self, channel: int, value: int, ai_type: str) -> No
...
+                _image['Image'].update(_parameters)
+                
+        if sharpen is not None:
+            if not isinstance(sharpen, int):
+                raise InvalidParameterError(f"set_image: sensitivity '{sharpen}' is not integer")
+            if sharpen < 0 or sharpen > 255:
+                raise InvalidParameterError(f"set_image: sharpen {sharpen} not in range 0...255")
+            else:
+                _parameters={"sharpen":sharpen}
+                _image['Image'].update(_parameters)
```
Comment (starkillerOG) [inline reolink_aio/api.py:3216]:
```suggestion
        if bright is not None:
            if not isinstance(bright, int):
                raise InvalidParameterError(f"set_image: bright '{bright}' is not integer")
            if bright < 0 or bright > 255:
                raise InvalidParameterError(f"set_image: bright {bright} not in range 0...255")
            else:
                _image['Image']['bright'] = bright

        if contrast is not None:
            if not isinstance(contrast, int):
                raise InvalidParameterError(f"set_image: contrast '{contrast}' is not integer")
            if contrast < 0 or contrast > 255:
                raise InvalidParameterError(f"set_image: contrast {contrast} not in range 0...255")
            else:
                _image['Image']['contrast'] = contrast

        if saturation is not None:
            if not isinstance(saturation, int):
                raise InvalidParameterError(f"set_image: saturation '{saturation}' is not integer")
            if saturation < 0 or saturation > 255:
                raise InvalidParameterError(f"set_image: saturation {saturation} not in range 0...255")
            else:
                _image['Image']['saturation'] = saturation
                
        if hue is not None:
            if not isinstance(hue, int):
                raise InvalidParameterError(f"set_image: hue '{hue}' is not integer")
            if hue < 0 or hue > 255:
                raise InvalidParameterError(f"set_image: hue {hue} not in range 0...255")
            else:
                _image['Image']['hue'] = hue
                
        if sharpen is not None:
            if not isinstance(sharpen, int):
                raise InvalidParameterError(f"set_image: sharpen '{sharpen}' is not integer")
            if sharpen < 0 or sharpen > 255:
                raise InvalidParameterError(f"set_image: sharpen {sharpen} not in range 0...255")
            else:
                _image['Image']['sharpen'] = sharpen

```

**Comment by starkillerOG** [comment]
@zerinrc I made some small tweaks to make sure it passes all the styling checks.
I think it looks good now.

I will publish a new version of reolink-aio in a few days, which will then include this new code.

Thank you very much for your contribution, looking forward to your next pull request.

**Comment by zerinrc** [comment]
Thanks to you, for this library.
:)

**Comment by mnpg** [comment]
hi,
for these parameters, are they depend on Camera GetAbility command?
In my case (E1-OutDoor), the hue parameter isn't possible or workable (Getability : "abilityChn": >  "ispHue" ver=0)
Regards

**Comment by mnpg** [comment]
Hi @starkillerOG,
In that PR, i ain't seen some capabilities dependances in the code for these Isp settings.
For my E1-Outdoor, the Hue setting isn't present, and it's normal because in the GetAbility, ispHue ver parameter is set to 0.
All these settings depends on these capabilities : ispBright, ispContrast, ispSatruation (yes, it's the real name), ispSharpen and ispHue

**Comment by starkillerOG** [comment]
@mnpg I just added the check of the capabilities in commit: https://github.com/starkillerOG/reolink_aio/commit/21adca678dd4d568e1d8e37ac95b547c3f170bc7
Thanks for making me aware of this.

**Comment by starkillerOG** [comment]
@zerinrc reolink-aio 0.6.0 has just been released, which includes your new code, it is now ready to be used.

**Pull request merged**